package.json

```diff
@@ -25,31 +25,22 @@
     "sandbox:build": "rollup -c --environment SANDBOX && cd sandbox && npm run build",
     "dev": "concurrently --names build,sandbox \"rollup -c -w --environment SANDBOX\" \"cd sandbox && export REACT_APP_NONCE=321 && npm start\"",
     "build": "npm run format && npm run lint && npm run clean && rollup -c --environment BASE_CODE_MIN,STANDALONE,STANDALONE_MIN && echo \"Base Code:\" && cat distTest/baseCode.min.js",
-    "prepare": "cd sandbox && npm install",
+    "prepare": "husky install && cd sandbox && npm install",
     "prepublishOnly": "rimraf libEs5 libEs6 && babel src -d libEs5 --env-name npmEs5 && babel src -d libEs6 --env-name npmEs6",
     "checkthattestfilesexist": "./scripts/checkThatTestFilesExist.js"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "echo 'Running pre-commit scripts...' && npm run checkthattestfilesexist && lint-staged && npm run test:unit"
-    }
-  },
   "lint-staged": {
     "./*.js": [
-      "eslint --fix",
-      "git add"
```

Comment:
The new lint-staged handles staging the modified file, so this `git add` command is no longer necessary (and lint-staged reports a warning if it finds it here).